src/components/SettingsModal/SettingsModal.tsx

```diff
@@ -275,14 +282,17 @@ const SettingsModal: React.FC<SettingsModalProps> = ({ open, onClose }) => {
               `}
             >
               {slippageAlert && <AlertTriangle color='#ffa000' size={16} />}
-              <input
+              <NumericalInput
                 placeholder={(userSlippageTolerance / 100).toFixed(2)}
-                className={classes.settingsInput}
                 value={slippageInput}
+                fontSize={14}
+                fontWeight={500}
+                align='right'
+                color='rgba(212, 229, 255, 0.8)'
                 onBlur={() => {
                   parseCustomSlippage((userSlippageTolerance / 100).toFixed(2));
                 }}
-                onChange={(e: any) => parseCustomSlippage(e.target.value)}
+                onUserInput={(value) => parseCustomSlippage(value)}
```

Comment:
Is on user input a simplification so we don have to go through `e.Target`

can `value` be typed?

Reply:
Yes, the value is the string typed in the input